tests/conf/i18n/translationplaceholderutils.js

```diff
@@ -0,0 +1,32 @@
+const Handlebars = require('handlebars');
+const { fromMustacheStatementNode } = require('../../../conf/i18n/translationplaceholderutils');
+const TranslationPlaceholder = require('../../../conf/i18n/models/translationplaceholder');
+describe('fromMustacheStatementNode usage', () => {
+  it('standard usage', () => {
+    const template = `{{ 
+      translate 
+      phrase='result' 
+      pluralForm='results' 
+      count='resultCount' 
+      param1='param1'
+      param2='param2'
+      context='testing' 
+    }}`;
+
+    const indexOfFirstMustacheStatement = 0;
+    const mustacheStatement = Handlebars.parse(template).body[indexOfFirstMustacheStatement];
```

Comment:
Could we just have the mustache statement object hard coded instead of using Handlebars.parse to get it? I think it might simplify this test a little, and also make it more clear what is being taken in by `fromMustacheNode`.

Reply:
I could, but it would add at least 60 lines of code since the mustache statement object is so large. Is that worth it?

Reply:
hmm okay it's probably fine then!